Client/Application/LeanplumIntegration.swift

```diff
@@ -135,6 +135,12 @@ class LeanplumIntegration {
         }
     }
 
+    func track(eventName: LeanplumEventName, withInfo: String) {
+        if shouldSendToLP() {
+            Leanplum.track(eventName.rawValue, withInfo: withInfo)
```

Comment:
I see this in the Leanplum documentation:

```
+ (void)track:(NSString *)event withParameters:(NSDictionary *)params
```

which I think is a better alternative for `info`? Like we could send more structured event data

```
{"source":"menu"}
```